[DO NOT MERGE] Test fixture: editorial issues for dashboard review composer (#38672)
DataDog/documentation: content/en/metrics/faq/rollup-for-distributions-with-percentiles.md

Time aggregation `(.rollup(<aggregator>, <time interval>)` and how it works with distributions with percentile aggregations has changed.

> ## Does this update to rollup change the values of my existing queries on distributions with percentiles? 

Author: ```suggestion
test6
```

> No - your existing queries are unaffected. We removed the `.rollup <aggregator>` parameter in order to reduce confusion, since it had no effect on query results but caused user confusion for how the resulting value was calculated. 

Author: "In order to" is wordier than "to," the sentence mentions confusion twice, and the switch to first-person "We removed" introduces a voice inconsistent with the rest of the page.

```suggestion
No - your existing queries are unaffected. The `.rollup <aggregator>` parameter was removed because it had no effect on query results but caused confusion about how the resulting value was calculated.
```

> No - your existing queries are unaffected. We removed the `.rollup <aggregator>` parameter in order to reduce confusion, since it had no effect on query results but caused user confusion for how the resulting value was calculated. 

Author: "In order to" is wordier than "to," the sentence mentions confusion twice, and the switch to first-person "We removed" introduces a voice inconsistent with the rest of the page.

```suggestion
test3
```

> No - your existing queries are unaffected. We removed the `.rollup <aggregator>` parameter in order to reduce confusion, since it had no effect on query results but caused user confusion for how the resulting value was calculated. 

Author: `in order to` is wordier than plain `to`. The sentence also uses first person (`We removed`), and it says "confusion" twice in one sentence (`reduce confusion... caused user confusion`), which is redundant.

```suggestion
No - your existing queries are unaffected. The `.rollup <aggregator>` parameter has been removed because it had no effect on query results and caused confusion about how the resulting value was calculated.
```

## What has been updated for distributions with percentiles?

> You no longer need to utilize an additional time aggregator (as in `.rollup(avg)`) when you've already selected `p50`, `p75`, `p90`, `p95`, or `p99`. You only need to specify a rollup `<time interval>`, which easily determines the interval of time your data is aggregated over. 

Author: "Utilize" should be "use." "Easily determines" is an odd modifier placement — it implies the interval itself does something "easily," rather than the parameter making configuration easy.

```suggestion
You no longer need to use an additional time aggregator (as in `.rollup(avg)`) when you've already selected `p50`, `p75`, `p90`, `p95`, or `p99`. You only need to specify a rollup `<time interval>`, which determines the interval of time your data is aggregated over.
```

> You no longer need to utilize an additional time aggregator (as in `.rollup(avg)`) when you've already selected `p50`, `p75`, `p90`, `p95`, or `p99`. You only need to specify a rollup `<time interval>`, which easily determines the interval of time your data is aggregated over. 

Author: "Utilize" should be "use." "Easily determines" is an odd modifier placement — it implies the interval itself does something "easily," rather than the parameter making configuration easy.

```suggestion
You no longer need to use an additional time aggregator (as in `.rollup(avg)`) when you've already selected `p50`, `p75`, `p90`, `p95`, or `p99`. You only need to specify a rollup `<time interval>`, which determines the interval of time your data is aggregated over.
```

> You no longer need to utilize an additional time aggregator (as in `.rollup(avg)`) when you've already selected `p50`, `p75`, `p90`, `p95`, or `p99`. You only need to specify a rollup `<time interval>`, which easily determines the interval of time your data is aggregated over. 

Author: `utilize` is a longer synonym for `use` with no added meaning here. `easily` is unnecessary filler that doesn't add information.

```suggestion
You no longer need to specify an additional time aggregator (as in `.rollup(avg)`) when you've already selected `p50`, `p75`, `p90`, `p95`, or `p99`. You only need to specify a rollup `<time interval>`, which determines the interval of time your data is aggregated over.
```

_Example: Suppose you're graphing on a dashboard timeframe of the Past 4 hours, p99:distribution.rollup(60)._
Datadog merges the sketch data server-side to represent a rollup interval of a minute (for every minute within the dashboard timeframe of 4 hours). After the rollup is calculated, Datadog calculates a 99th percentile value for each minute interval.

## Rollup Interval Best Practices

> It is recommended that a rollup interval be chosen based on the range being queried. As described above, the rollup `<aggregator>` parameter no longer affects distributions queried with percentiles, so only the `<time interval>` needs to be specified.

Author: test1

**Suggested fix:** test2

> It is recommended that a rollup interval be chosen based on the range being queried. As described above, the rollup `<aggregator>` parameter no longer affects distributions queried with percentiles, so only the `<time interval>` needs to be specified.

Author: test1

**Suggested fix:** test2

> It is recommended that a rollup interval be chosen based on the range being queried. As described above, the rollup `<aggregator>` parameter no longer affects distributions queried with percentiles, so only the `<time interval>` needs to be specified.

Author: The new `## Rollup Interval Best Practices` heading is title case and not phrased as a question, unlike every other heading on this FAQ page. Its content restates what's already covered in the `Why can't I specify a rollup/time aggregator anymore` section and the second paragraph of the page.

**Suggested fix:** Merge this content into the existing section that already covers the same point, or if it stays separate, rephrase the heading as a question in sentence case consistent with the rest of the page.
